.rubocop.yml

```diff
@@ -33,3 +33,5 @@ Metrics/BlockLength:
   Enabled: false
 Gemspec/RequiredRubyVersion:
   Enabled: false
+Layout/LineLength:
+  Enabled: false
```

Comment:
It would be better to disable it locally: https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code or even better comply with this rule.

I believe you can use autocorrection to fix issues: https://docs.rubocop.org/rubocop/cops_layout.html#layoutlinelength

The default value is 120, I don't mind if we configure 140.

Reply:
OK that is fine for me, 120 is a little bit short for displays with a size of 27' ;)

Reply:
I find 120 to be a good value. It's not about the size of the monitor. Lines longer than 120 are very hard to read on GitHub and in split views. I think keeping it at 120 encourages a good practice of not having run-on statements.